resource-definitions/template-driver/annotations/annotations.yaml

```diff
@@ -0,0 +1,21 @@
+apiVersion: entity.humanitec.io/v1b1

```

Comment:
Suggesting to add a comment to show what this file does. This will especially help our AI bot to better classify the code sample. It is a recommended practice by our AI provider.

```suggestion
# This Resource Definition shows how to add annotations to the Kubernetes service object using the Template Driver
apiVersion: entity.humanitec.io/v1b1

```